packages/web3-validator/src/types.ts

```diff
@@ -0,0 +1,16 @@
+import { ErrorObject } from 'ajv';
+import { AbiParameter } from './private_types';
+
+export { JSONSchemaType } from 'ajv';
+export { DataValidateFunction, DataValidationCxt } from 'ajv/dist/types';
```

Comment:
```suggestion
import { DataValidateFunction, DataValidationCxt, ErrorObject, JSONSchemaType } from 'ajv';
import { AbiParameter } from './private_types';

export {  } from 'ajv';
```

I feel like `eslint` should've complained about this. I'm not sure if `DataValidateFunction, DataValidationCxt` is importable like above

Reply:
It's not imported, its exported here. For the users to of the package, they may need those types for extending nay some validation.